setup.py

```diff
@@ -27,7 +27,7 @@ def get_version(package):
 
 def get_packages(package):
     """
-    Return root package and all sub-packages.
+    Return root package and all sub-packages.develop
```

Comment:
definitively not, I wondered where this went to, but I couldn't spot it in the commit - should have looked harder; thanks for the catch.